starter-core/src/main/java/io/micronaut/starter/feature/function/gcp/template/gcpFunctionJavaJunit.rocker.raw

```diff
@@ -16,9 +16,20 @@ import io.micronaut.gcp.function.http.*;
 public class @project.getClassName()FunctionTest {
 
     @@Test
-    public void testFunction() throws Exception {
+    public void testGet() throws Exception {
         try (HttpFunction function = new HttpFunction()) {
-            GoogleHttpResponse response = function.invoke(HttpMethod.GET, "/@project.getPropertyName()");
+            GoogleHttpResponse response = function.invoke(HttpMethod.GET, "/");
+            assertEquals(HttpStatus.OK, response.getStatus());
+        }
+    }
+
+    @@Test
+    public void testPost()throws  Exception{
+        try (HttpFunction function = new HttpFunction()) {
+            SampleInputMessage sampleInputMessage = new SampleInputMessage();
+            sampleInputMessage.setName("Test Name");
+            HttpRequest request = HttpRequest.POST("/", sampleInputMessage).contentType(MediaType.APPLICATION_JSON_TYPE);
```

Comment:
`.contentType(MediaType.APPLICATION_JSON_TYPE)` is unnecessary. Micronaut defaults to `application/json`